my-app/src/redux/login/reducers.js

```diff
@@ -0,0 +1,21 @@
+import { actions } from './actions';
+
+const initialState = {
+  Authorization: '',
+  logged: false
+};
+
+function LogInReducer(state = initialState, action) {
+  switch (action.type) {
+    case actions.LOG_IN: {
+      return {
+        Authorization: action.payload.Authorization,
+        logged: true
```

Comment:
`logged` is redundant, since we can infer if an user is logged or not via `authorization`